docs/meetups/philly.md

```diff
@@ -0,0 +1,15 @@
+Write the Docs Philly
+===========================
+
+## About Us
+
+<img src="/_static/img/logos/writethedocsphl-logo.png" style="margin:0 0 20px 20px;width:200px;max-width:100%;float:right;" alt="Write the Docs Philly Logo" />
```

Comment:
@ericholscher I made the NYC one in Gimp and then edited it for the Philly version. Here's the Gimp file: https://drive.google.com/open?id=0B0Mvprr0i5d_Y0Q3Qzk2YTZKR1U